torch/csrc/jit/codegen/cuda/executor_utils.cpp

```diff
@@ -232,6 +232,48 @@ EvaluationContext bindInputs(
   return eval_context;
 }
 
+EvaluationContext bindInputs(
```

Comment:
Should we refactor to get rid of the previous bindInputs function?

Reply:
CC @tlemo

Reply:
I agree, we should try to avoid the duplication. (The whole binding & evaluation of Kernel IR expressions will likely see significant changes in the near future, so I wouldn't worry to much about having a perfect solution at this point)